workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -98,13 +118,40 @@ public static Request CreateRequest(Entity entity, EntityId? entityId = null, ui
                 };
             }
 
+            /// <summary>
+            ///     An object that represents the response of a CreateEntity command from the SpatialOS runtime.
```

Comment:
I think the object is the response, not a representation of it.

Reply:
Or maybe contains it

Reply:
Yeah it contains it + other data (although its flattened). 

Although I feel conceptually "represents" is a nice way of thinking about it